2-mandatory/1-create-functions.js

```diff
@@ -24,17 +21,22 @@ Write a function that:
 - Removes any forward slashes (/) in the strings.
 - Makes the strings all lowercase.
 */
-function tidyUpString() {
-}
+const modifyStr = (names) => names.trim().replace("/", "").toLowerCase();
+
+const tidyUpString = (arrOfString) => arrOfString.map(modifyStr);
```

Comment:
Very nice Laleh! I always like to separate my functions so it is easier to read, modify and understand! If you wanted to combine these two, you could do: 

`const tidyUpString = (arr) => arr.map(e => e.trim().replace("/", "").toLowerCase());`